public/locales/en/translations.json

```diff
@@ -411,6 +412,25 @@
           "value": "Value",
           "valuePlaceholder": "Enter value"
         },
+        "actions":{
+          "actions": "Actions",
+          "action": "Action",
+          "emailActionLabel": "Send email",
+          "emailActionVal": "Email",
+          "noActionLabel": "No action",
+          "noActionVal": "none",
+          "addAction": "Add action",
+          "select": "Choose an action",
+          "selectionPlaceholder": "Select action",
+          "emailAddresses": "Email addresses",
+          "enterEmail": "Press enter to add email recipient",
+          "emailComments": "Email comments",
```

Comment:
Nit: EmailComm vs EmailComments, maybe pick one for consistency?

Reply:
Fixed to emailcomments